src/util.ts

```diff
@@ -37,43 +37,153 @@ const request = r.defaults({
   },
 });
 
-const util = module.exports;
+export interface PromisifyAllOptions {
```

Comment:
Maybe `PromisifyAllOptions` can extend `PromsifyOptions`? That way the `as any` in `promisifyAll` is unneeded.